atom/browser/common_web_contents_delegate.cc

```diff
@@ -208,6 +209,8 @@ void CommonWebContentsDelegate::SetOwnerWindow(
   int32_t id =
       api::Window::TrackableObject::GetIDFromWrappedClass(owner_window);
   if (id > 0) {
+    SessionID sessionID;
```

Comment:
Is `sessionID` not being used? `TabHelper::SetWindowId` will eventually set SessionID and use it so I think we can remove it here.